src/heap_ptmalloc_2_27.cpp

```diff
@@ -1816,7 +1816,7 @@ static bool fill_heap_block(struct ca_heap* heap, address_t addr, struct heap_bl
 	struct malloc_chunk achunk;
 	size_t chunksz;
 	size_t size_t_sz = sizeof(INTERNAL_SIZE_T);
-	size_t mchunk_sz = sizeof(struct malloc_chunk);
+	size_t mchunk_sz = sizeof(struct malloc_chunk_s);
```

Comment:
The code here only cares about the size(previous size, size), but the memory content is stored in `malloc_chunk` in line 1816. Should it be stored into the `malloc_chunk_s` to be consistent?

Reply:
Yes, my initial thought is that it doesn't matter. But it is better for the sake of readability.